doc/changes/latest.inc

```diff
@@ -97,3 +97,5 @@ API and behavior changes
 - When creating BEM surfaces via :func:`mne.bem.make_watershed_bem` and :func:`mne.bem.make_flash_bem`, the ``copy`` parameter now defaults to ``True``. This means that instead of creating symbolic links inside the FreeSurfer subject's ``bem`` folder, we now create "actual" files. This should avoid troubles when sharing files across different operating systems and file systems (:gh:`10531` by `Richard Höchenberger`_)
 
 - The ordering of channels returned by :func:`mne.io.read_raw_nirx` is now ordered by channel name, rather than the order provided by the manufacturer. This enables consistent ordering of channels across different file types (:gh:`10555` by `Robert Luke`_)
+
+- For :func:`mne.viz.plot_topomap`, :func:`mne.viz.plot_evoked_topomap`, :func:`mne.viz.plot_arrowmap`, :func:`mne.viz.plot_ica_components`, :meth:`mne.Covariance.plot_topomap`, :meth:`mne.Evoked.plot_topomap`, :meth:`mne.Evoked.animate_topomap`, :meth:`mne.decoding.CSP.plot_patterns`, :meth:`mne.Projection.plot_topomap` and :meth:`mne.preprocessing.ICA.plot_components` the topomap image interpolation was previously a cubic interpolation but now can be ``'linear'`` and ``'nearest'`` as well. There was a subsequent matplotlib plotting interpolation after the original interpolation from points into an image, now ``'image_interp'`` controls the first interpolation (the plotting interpolation can be set afterward using ``im.set_interpolation``) (:gh:`10617` by `Alex Rockhill`_)
```

Comment:
I'd say it this way:
```
``'nearest'`` as well. Unless ``image_interp='nearest'`` is passed mne uses
subsequent matplotlib bilinear interpolation to make the interpolated
image smoother. Previously, ``image_interp`` controlled this second interpolation
step, but it was decided to be confusing. Now ``image_interp`` is responsible for
the main interpolation. To get the old behavior (...)
```
(sorry, the I didn't check the line lengths so they are probably incorrect in my suggestion)